kubectl-fdb/cmd/remove_process_group_test.go

```diff
@@ -33,6 +33,7 @@ var _ = Describe("[plugin] remove process groups command", func() {
 	When("running remove process groups command", func() {
 		When("removing process groups from a cluster", func() {
 			BeforeEach(func() {
+				cluster.Spec.ProcessGroupIDPrefix = ""
```

Comment:
This one is questionable whether or not I should be fixing all of the test cases or setting prefix to nothing.  It felt fine to be since the tests are setup this way and are consistent with no prefix, and it's not bad to test the empty prefix case too.  
I'll have some coverage on the non-empty prefix case in https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1930/files but let me know if it'd be more comfortable to have more test with the prefix set.

Reply:
It's fine to keep the tests as they are, we can change them later if we want.